### ISSUE TITLE
Json wire bad handling of unicode characters 

### ISSUE THREAD

**Issue by RobAustin** [OpenHFT/Chronicle-Wire#604]
```
[ {"msg":"8=FIX.4.1|9=61|35=A|34=1|49=EXEC|52=20121105-23:24:06|56BANZAI|98=0|108=30|10=003","cause":"For input string: \"56BANZAI\x0198\"","status":"parse-error"},{"msgType":"A","msgType_Desc":"Logon","senderCompID":"BANZAI","targetCompID":"EXEC","msgSeqNum":1,"sendingTime":"2012-11-05T23:24:06","encryptMethod":0,"heartBtInt":30,"encryptMethod_Desc":"NONE OTHER","msg":"8=FIX.4.1|9=61|35=A|34=1|49=EXEC|52=20121105-23:24:06|56BANZAI|98=0|108=30|10=003|8=FIX.4.1|9=61|35=A|34=1|49=BANZAI|52=20121105-23:24:06|56=EXEC|98=0|108=30|10=003|"} ]
```

We should not encode to this:

> characters that must be escaped: quotation mark, reverse solidus, and the control characters (U+0000 through U+001F).

![image](https://user-images.githubusercontent.com/6506056/204561455-ed2b02b4-7adf-46d2-82c3-a224201f511f.png)

but rather follow the standard here:

see RFC 7159: The JavaScript Object Notation (JSON) Data Interchange Format
https://www.rfc-editor.org/rfc/rfc7159#section-7

**Comment by RobAustin** [comment]
relates to https://github.com/ChronicleEnterprise/Chronicle-Platform/issues/329

**Comment by maxim-ponomarev** [comment]
Please, prioritise this issue.
It causes https://github.com/ChronicleEnterprise/Chronicle-Platform/issues/420.
Reproduced in https://github.com/OpenHFT/Chronicle-Wire/commit/90a6866ee9e1120c63a59ac48ddb6dd033fdf713

**Comment by hft-team-city** [comment]
Released in [Chronicle-Wire-2.24ea7](https://github.com/OpenHFT/Chronicle-Wire/releases/chronicle-wire-2.24ea7), [BOM-2.24ea38](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.24ea38)
